### ISSUE TITLE
not able to reproduce this demo with latest GA (4.4.0.Final)

### ISSUE THREAD

**Issue by rafaeltuelho** [stianst/keycloak-containers-demo#1]
It appears this demo depends on some upcoming version... I changed to `4.4.0.Final` but the build breaks at

```
import org.keycloak.crypto.SignatureProvider;
import org.keycloak.crypto.SignatureVerifierContext;
```
Is there any workaround to reproduce this demo for now? Maybe disabling some feature?
When `4.5.0` will be released?

**Comment by stianst** [comment]
4.5 changes are related to client scopes and signatures. You can build Keycloak from source locally to get the required dependencies, wait for Keycloak 4.5 to be released later this week or remove the token-verifier module from being built/deployed (you won't be able to verify tokens with /jwt endpoint, but everything else will work as it's using a Docker image built from Keycloak master).

**Comment by rafaeltuelho** [comment]
Thanks @stianst ! I'll try without the `token-verifier` for now.

**Comment by stianst** [comment]
Sorry for it taking so long, but the demo is fixed now and updated to Keycloak 9.0.2
